tests/integration/test_installation.py

```diff
@@ -166,11 +168,18 @@ def test_running_real_remove_backup_groups_job(ws, installation_ctx):
 
     installation_ctx.deployed_workflows.run_workflow("remove-workspace-local-backup-groups")
 
-    @retried(on=[NotFound], timeout=timedelta(seconds=120))
-    def wait():
-        ws.groups.get(ws_group_a.id)
-
-    wait()
+    # pylint: disable=duplicate-code
```

Comment:
Accept duplicate code as it is only used twice. If this is not wanted, it could be moved into a fixture.

Reply:
no duplicate code :) there's a smaller change possible.